jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -308,6 +309,7 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
     ReactDOM.render(
       <HeaderHelp
         appState={this.state}
+        changeState={this.state.changeState.bind(this.state)}
```

Comment:
see `_bindRenderUtilities` in init.js for a pre-bound version of this.

Reply:
done - and all other instances of `this.state.changeState.bind(` in `component-renderers-mixin.js`